File: app/src/main/java/com/mhammad/movieapp/ui/MovieViewModel.java
package com.mhammad.movieapp.ui;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import com.mhammad.movieapp.pojo.MovieModel;

public class MovieViewModel extends ViewModel {
    public MutableLiveData<String> movieNameMLV = new MutableLiveData<>();
    public LiveData<String> getMovieName(){
        movieNameMLV.setValue(getMovieFromDB().movieName);
        return movieNameMLV;
    }

    private MovieModel getMovieFromDB()
    {
        return new MovieModel(1, "CUKUR", "17/9");
    }

}

File: app/src/main/java/com/mhammad/moviesappkt/ui/MovieViewModel.kt
package com.mhammad.moviesappkt.ui

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.mhammad.moviesappkt.pojo.MovieModel

class MovieViewModel: ViewModel() {
    private var movieName = MutableLiveData<String>()
    fun getMovieName() : MutableLiveData<String>
    {
        movieName.value = getMovie().movieName
        return movieName
    }
    fun getMovie(): MovieModel{
        return MovieModel("CUKUR", 1, "17/9")
    }
}